patches/obs-glcapture.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/src/obs-glcapture b/src/obs-glcapture
+index 2efcf51..be7e43c 100755
+--- a/src/obs-glcapture
++++ b/src/obs-glcapture
+@@ -1,6 +1,7 @@
+ #!/bin/sh
+ 
+ OBS_GLCAPTURE_LIB="libobs_glcapture.so"
++LIBDIR="/usr/lib/extensions/vulkan/OBSVkCapture/\$LIB"
+ 
+ if [ "$#" -eq 0 ]; then
+     programname=`basename "$0"`
+@@ -11,4 +12,5 @@ if [ "$#" -eq 0 ]; then
+ fi
+ 
+ LD_PRELOAD="${LD_PRELOAD}:${OBS_GLCAPTURE_LIB}"
+-exec env LD_PRELOAD="${LD_PRELOAD}" "$@"
++LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:${LIBDIR}"
```

Comment:
Nitpick: check if the current `LD_LIBRARY_PATH` is empty, append `${LIBDIR}` to the current one if it isn't, replace the current one if it is. E.g.
```bash
if [ -z "${LD_LIBRARY_PATH}" ]; then
  export LD_LIBRARY_PATH="${LIBDIR}"
else
  export LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:${LIBDIR}"
fi
```

Reply:
Done

Reply:
Does it still work? :-)
Not a blocker for the flatpak, just a suggestion: if you're familiar with cmake, consider configuring the `obs-glcapture` script via cmake at build time, putting in it the actual install prefix (instead of hardcoded `/usr/lib/extensions/vulkan/OBSVkCapture`). This way these changes could be submitted upstream.

Reply:
Yes, it still works, otherwise I wouldn't have committed it :smile: Not too familiar with CMake unfortunately, but maybe I'll take a look at it sometime and make a PR